src/roblox_api/mod.rs

```diff
@@ -10,6 +11,7 @@ use crate::{
     value::{Color3Value, Vector3Value, Vector3int16Value},
 };
 
+use cframe::*;
```

Comment:
This use is kinda weird. Can we use only the part that we use instead of a glob import?